freeipa/src/main/java/com/sequenceiq/freeipa/converter/cloud/StackToCloudStackConverter.java

```diff
@@ -109,8 +112,14 @@ public CloudStack convertForTermination(Stack stack, String instanceId) {
     public CloudStack convert(Stack stack, Collection<String> deleteRequestedInstances) {
         Image image = imageConverter.convert(imageService.getByStack(stack));
         Optional<CloudFileSystemView> fileSystemView = buildFileSystemView(stack);
-        List<Group> instanceGroups = buildInstanceGroups(Lists.newArrayList(stack.getInstanceGroups()), stack.getStackAuthentication(),
-                deleteRequestedInstances, stack.getCloudPlatform(), image.getImageName(), fileSystemView, stack.getEnvironmentCrn());
+        List<Group> instanceGroups = buildInstanceGroups(
+                stack,
+                Lists.newArrayList(stack.getInstanceGroups()),
+                deleteRequestedInstances,
+                stack.getCloudPlatform(),
+                image.getImageName(),
+                fileSystemView,
+                stack.getEnvironmentCrn());
```

Comment:
Nit: Since we are passing `stack` anyway, `stack.getCloudPlatform()` and `stack.getEnvironmentCrn()` could be also removed as explicit arguments.